_docs/_latest/configure/inventory/gsuite.md

```diff
@@ -70,3 +70,68 @@ If you are running Forseti on GCP and made any changes to the above values,
 you will need to copy the `conf` file to the Cloud Storage bucket. For more
 information, see
 [Moving configuration to Cloud Storage]({% link _docs/latest/configure/general/index.md %}).
+
+## Troubleshooting
+
```

Comment:
It'd be nice to open by saying something along the lines - 'Below are the common errors and the steps to be taken to resolve the errors.`

Reply:
Great suggestion, done.